models/git/branch_list.go

```diff
@@ -63,30 +64,83 @@ func (branches BranchList) LoadPusher(ctx context.Context) error {
 	return nil
 }
 
+func (branches BranchList) LoadRepo(ctx context.Context) error {
+	ids := container.Set[int64]{}
```

Comment:
Now we can should use `container.FilterSlice` to get repoIDs

Reply:
Done.